p2p/DEVELOPMENT.md

```diff
@@ -18,7 +18,7 @@ library.
 ## BaseService
 
 - If your service needs to run coroutines in the background, you should use the `BaseService.run_task()` method and
-  ensure they exit when `is_running` is False or when the cancel token is triggered.
+  ensure they exit when `is_operational` is False or when the cancel token is triggered.
```

Comment:
Just making a note that the `BaseService` class is probably at a stage where we could document it or potentially start the process of extracting it into a stand-alone `asyncio-service` library or something like that.